Controllers/OperatorsController.cs

```diff
@@ -29,4 +29,11 @@ public IEnumerable<WeatherForecast> Get()
         })
         .ToArray();
     }
+
+    [HttpGet]
+    [Route("subtracao")]
+    public double Subtracao(double n1, double n2)
```

Comment:
Metados são criados com a primeira letra maiúscula
```suggestion
    public double Subtracao(double n1, double n2)
```